src/third_party/catch/include/helpers.hpp

```diff
@@ -19,7 +19,7 @@
         REQUIRE(!OBJECT.NAME());                     \
     }                                                \
                                                      \
-    SECTION("has a method to set the upsert") {      \
+    SECTION("has a method to set the NAME") {        \
```

Comment:
I was really confused when my broken read_concern test caused the "upsert" test failed :)